cluster-autoscaler/provisioningrequest/checkcapacity/condition.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package checkcapacity
+
+import (
+	"time"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1"
+	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/provreqwrapper"
+	"k8s.io/klog/v2"
+)
+
+const (
+	defaultReservationTime = 10 * time.Minute
+	defaultExpirationTime  = 7 * 24 * time.Hour // 7 days
+)
+
+const (
+	capacityIsNotFoundReason = "Capacity is not found"
```

Comment:
please make the reasons CamelCase, i.e.
```
CapacityWasNotFound
CapacityFound
FailedToBookCapacity
```

Reply:
Also please add Reason suffix to all variable names

Reply:
Sure. Is this a naming convention for reasons?

Reply:
But I think we should use present tense, because we add time and describe what was happened in that time.

Reply:
We can use the present tense.
But please use the actual reason strings camel case:
```
"Capacity is not found" -> "CapacityIsNotFound"
```